docs/standards/smart-contracts/lsp7-digital-asset.md

```diff
@@ -163,8 +170,77 @@ _Triggers the **[RevokedOperator](#revokedoperator)** event when an address get
 
 :::
 
+### increaseAllowance
+
+:::info
+This is a non-standard function, not part of the `ILSP7DigitalAsset` interface. It exists to be used as a prevention mechanism against the double spending allowance vulnerability.
+:::
+
+```solidity
+function increaseAllowance(address operator, uint256 addedAmount) external nonpayable
```

Comment:
whats the point of nonpayable modifier

Reply:
it's added by default by hardhat-dodoc. I can remove it.